rosa/terraform/outputs.tf

```diff
@@ -0,0 +1,135 @@
+# SPDX-FileCopyrightText: 2024 SAP edge team
```

Comment:
The current outputs.tf contains too many irrelevant outputs that don't help users
  access the cluster. Pls only include the access related info to the cluster.